### ISSUE TITLE
Node.js 12 actions and `set-output` are deprecated

### ISSUE THREAD

**Issue by jkunze** [martinthomson/i-d-template#378]
I don't know if this is an issue for i-d-template, but I'm getting some warnings from the Update Editor's Copy workflow:

1. Node.js 12 actions are deprecated.
2. The `set-output` command is deprecated and will be disabled soon.

Is there a recommended way of handling this?

**Comment by dwaite** [comment]
There appear to have been fixes for this in the internet-draft-template repo with commit f39cb0 - perhap this would be resolved if you update the GitHub actions in your repository and try again?

**Comment by jkunze** [comment]
Thanks. I've run "Update generated files", which nicely corrected the out-of-date email addresses in CODEOWNERS. I'm now waiting on the IETF datatracker folks to see if my previous submission needs to be cleared out.

**Comment by martinthomson** [comment]
Running the file update action won't fix the actions because actions can't update actions without extra permissions.  You'll need to do that manually, either by copying over the new files or by running `make update-files` in a clone of the repository.

**Comment by jkunze** [comment]
Great! I ran `make update-files`, which looks good far. Does this functionality replace what I found in .github/workflows/update.yml (maybe I should remove that file)?

**Comment by martinthomson** [comment]
`update.yml` is convenient for updating other stuff (like the readme and the codeowners file).  Keep it if you want to use it.
